packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,204 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized, useLocalization } from '@fluent/react';
+import { RouteComponentProps } from '@reach/router';
+import React, { useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from '@reach/router';
+import { HomePath } from '../../constants';
+import {
+  logViewEvent,
+  settingsViewName,
+  usePageViewEvent,
+} from '../../lib/metrics';
+import { useAccount, useAlertBar } from '../../models';
+import { InputText } from '../../components/Settings/InputText';
+import HeaderContinueTo from '../../components/App/HeaderContinueTo';
+import WarningMessage from '../../components/App/WarningMessage';
+import RememberPassword from '../../components/App/RememberPassword';
+
+// --forceEmail-- is a hint to the signup page that the user should not
+// be given the option to change their address
+// can be triggered by clicking on "Forgot password?" from /force_auth?email=user@domain.com
+// where the provided email is a registered account
+// (e.g., if coming from force_auth or a relying party)
+
+// --canGoBack-- determines if the user can navigate back to an fxa entrypoint
+// unsure when this might be false
+
+// --serviceName-- is the relying party
+
+// '?email=username@domain.com&reset_password_confirm=false' query params
+// skips reset_password (and warnings) and goes directly to confirm_reset_password
+
+// TODO verify if these are passed as props or obtained a different way
+export type ResetPasswordProps = {
+  serviceName?: string;
+  forceEmail?: string;
+  canGoBack?: boolean;
+};
+
+type FormData = {
+  email: string;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPassword = ({
+  serviceName,
+  forceEmail,
+  canGoBack,
+}: ResetPasswordProps & RouteComponentProps) => {
+  usePageViewEvent('settings.reset-password', {
+    entrypoint_variation: 'react',
+  });
+
+  const [email, setEmail] = useState<string>(forceEmail || '');
+  const [emailErrorText, setEmailErrorText] = useState<string>();
+  const [isFocused, setIsFocused] = useState(false);
+  const alertBar = useAlertBar();
+  const account = useAccount();
+  const { l10n } = useLocalization();
+  const navigate = useNavigate();
+  const pageId: string = 'reset-password';
+  const onFocusMetricsEvent = 'reset-password.engage';
+
+  const { handleSubmit } = useForm<FormData>({
+    mode: 'onBlur',
+    criteriaMode: 'all',
+    defaultValues: {
+      email: '',
```

Comment:
Should this be whatever `email` is?

```
defaultValues: {
  email,
}
```

Reply:
@LZoog I'd set the default value to an empty string because of this comment in `reset_password.js`:
```
    // The email field is not pre-filled for the reset_password page,
    // but if the user enters an email address, the entered email
    // address should be propagated back to the signin page. If
    // the user enters no email and instead clicks "Remember password?"
    // immediately, the /signin page should have the original email.
    // See https://github.com/mozilla/fxa-content-server/issues/5293.
```
I'm not sure _why_ we don't prefill the email though - seems like it would be a better experience to do as you suggest.